### ISSUE TITLE
Accessibility - iOS - French - VoiceOver Does Not Read Individual Steps on Main Steps Screen

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#106]
While using VoiceOver to read the individual steps on the main Steps screen for French, they are not getting read.
